build/docker/docker-compose-test.yml

```diff
@@ -0,0 +1,112 @@
+version: "3.7"
+
+services:
+  test-db:
+    image: $DOCKER_REGISTRY/data:latest
+    container_name: test-db
+    hostname: test-db
+    environment:
+      PROFILE: $PROFILE
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PGPASSWORD: postgres
+    ports:
+      - "5432:5432"
+    command: ["postgres", "-c", "config_file=/etc/postgresql/postgresql.conf"]
+    networks:
+      default:
+        aliases:
+          - test-db.$PROJECT_NAME_SHORT.local
+
+  data:
+    image: $DOCKER_REGISTRY/data:latest
+    container_name: data
+    hostname: data
+    environment:
+      PROFILE: $PROFILE
+      POSTGRES_DB: postgres
```

Comment:
Please, can we use the following variable names:
```
DB_HOST
DB_PORT
DB_NAME
DB_USERNAME
DB_PASSWORD
```
They are already defined and used in the image.